src/screens/repo/screens/build/menu.js

```diff
@@ -79,6 +114,26 @@ export default class BuildMenu extends Component {
 									</button>
 								)}
 							</li>
+							<li>
```

Comment:
This `<li>` item will be empty when status =! success, maybe we should move in condition?

Reply:
yes， you are right